test-framework/devmode-test-utils/src/main/java/io/quarkus/test/devmode/util/DevModeTestUtils.java

```diff
@@ -16,20 +16,24 @@
 
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.IOUtils;
-import org.jutils.jprocesses.JProcesses;
-import org.jutils.jprocesses.model.ProcessInfo;
 
 public class DevModeTestUtils {
 
     public static void killProcesses(String... cmdParts) {
-        for (ProcessInfo pi : JProcesses.getProcessList()) {
-            for (String part : cmdParts) {
-                if (pi.getCommand().contains(part)) {
-                    JProcesses.killProcess(Integer.parseInt(pi.getPid()));
-                    break;
-                }
-            }
-        }
+        ProcessHandle.allProcesses()
+                .filter(processHandle -> {
+                    String cmdLine = processHandle.info().commandLine().orElse(null);
```

Comment:
Just a personal thing I guess, but I would do the filtering like this (might contain a quirk, haven't compiled):
```
                .filter(processHandle -> processHandle.info().commandLine()
                    .map(line -> cmdParts.stream().anyMatch(line::contains))
                    .orElse(false))
```

Reply:
How about:
```java
    public static void killProcesses(String... cmdParts) {
        Stream<String> cmdPartsStream = Arrays.stream(cmdParts);
        ProcessHandle.allProcesses()
                .filter(processHandle -> processHandle.info().commandLine().stream()
                        .anyMatch(line -> cmdPartsStream.anyMatch(line::contains)))
                .forEach(ProcessHandle::destroyForcibly);
    }
``` ?

Reply:
Perfection!

Reply:
![image](https://user-images.githubusercontent.com/54133/114778641-7ec6d500-9d4b-11eb-9f59-2bd378807f82.png)